packages/create-plugin/templates/common/_package.json

```diff
@@ -22,7 +22,7 @@
     "@grafana/e2e": "{{ grafanaVersion }}",
     "@grafana/e2e-selectors": "{{ grafanaVersion }}",{{/unless}}
     "@grafana/eslint-config": "^7.0.0",{{#if usePlaywright}}
-    "@grafana/plugin-e2e": "^0.24.0",{{/if}}
+    "@grafana/plugin-e2e": "^0.27.0",{{/if}}
```

Comment:
I'm under the impression that we're not gonna release these changes until 1.0.1 is out the door?

Reply:
No you're right! It's just that one of the specs depend on changes released in 0.27.0, so needed to bump to be able to test and verify what was generated. We should point to 1.0.1 before merging to main. 👍